lua/lspconfig/gopls.lua

```diff
@@ -15,9 +15,7 @@ https://github.com/golang/tools/tree/master/gopls
 
 Google's lsp server for golang.
 ]],
-    default_config = {
-      root_dir = [[root_pattern("go.mod", ".git")]],
```

Comment:
The previous version here seems pretty solid, given that `go.mod` exists in the root of all modern `golang` projects.

Reply:
This is documentation only, it's autogenerated now and does not affect current behavior in any way.